packages/material-ui/src/internal/SwitchBase.js

```diff
@@ -165,11 +204,6 @@ SwitchBase.propTypes = {
    * The icon to display when the component is checked.
    */
   checkedIcon: PropTypes.node.isRequired,
-  /**
```

Comment:
We should still have this in the propTypes. I suspect it’s removed because we don’t extract kt from the props. Can you try to extract it from the props and save it in a variable classesProp? We don’t have to use it anywhere

Reply:
Cc @eps1lon

Reply:
I actually had removed this one manually, it doesn't seem that `yarn proptypes` touches this section of code.  Adding it back!